### ISSUE TITLE
No connection possible after beta2 installation

### ISSUE THREAD

**Issue by splitt3r** [nextcloud/desktop#592]
I installed the beta2 version over the latest daily build before it and now i can´t connect to my instance anymore. Besides that i updated my instance to version 14 RC1. All the other clients still work fine but the desktop client doesn´t follow the redirect after login. Btw. Avira Antivir complained about the QtWebEngine.dll (or sth. like that). My server logs dont´t show any error.

The tries are even listed as successful logins:
![login](https://user-images.githubusercontent.com/1869584/44682570-09941d80-aa44-11e8-92f7-b00c3ecd5ba4.png)

**Comment by camilasan** [comment]
Hi! Could you test this build - please make sure to have a clean configuration - https://download.nextcloud.com/desktop/daily/Windows/Nextcloud-2.5.0.61352-daily-20180904.exe?
It solved openssl/qtkeychain/credentials issues. 
Thanks!

**Comment by splitt3r** [comment]
Still doesn´t work. These are the logs:

```
[OCC::WebFlowCredentials::createQNAM 	Get QNAM
[OCC::AccessManager::createRequest 	6 "PROPFIND" "cloud.XXXXX.com/remote.php/webdav/" has X-Request-ID "XXXXXXX-XXXX-XXXX-XXXX-XXXXXXX"
[OCC::AbstractNetworkJob::start 	OCC::PropfindJob created for "://cloud.XXXXX.com" + "/" "OCC::OwncloudSetupWizard"
[OCC::WebFlowCredentials::slotFinished 	request finished
[OCC::AbstractNetworkJob::slotFinished 	QNetworkReply::NetworkError(ProtocolUnknownError) "Protocol \"\" is unknown" QVariant(Invalid)
[OCC::WebFlowCredentials::stillValid 	Still valid?
[OCC::WebFlowCredentials::stillValid 	QNetworkReply::NetworkError(ProtocolUnknownError)
[OCC::WebFlowCredentials::stillValid 	"Protocol \"\" is unknown"
[OCC::PropfindJob::finished 	PROPFIND of QUrl("") FINISHED WITH STATUS "ProtocolUnknownError Protocol \"\" is unknown"
[OCC::PropfindJob::finished 	*not* successful, http result code is 0 ""
[OCC::WebFlowCredentials::stillValid 	Still valid?
[OCC::WebFlowCredentials::stillValid 	QNetworkReply::NetworkError(ProtocolUnknownError)
[OCC::WebFlowCredentials::stillValid 	"Protocol \"\" is unknown"
```

Between switching the client i disconnected the account, delete %appdada%/(Roaming|Local)/Nextcloud and also the log files. Am i missing sth.?

**Comment by camilasan** [comment]
So you don't even connect via the web login?

**Comment by splitt3r** [comment]
Nope:

```
[OCC::WebFlowCredentials::stillValid 	QNetworkReply::NetworkError(AuthenticationRequiredError)
[OCC::WebFlowCredentials::stillValid 	"Error transferring https://cloud.XXXXX.com/remote.php/webdav/ - server replied: "
[OCC::PropfindJob::finished 	PROPFIND of QUrl("https://cloud.XXXXX.com/remote.php/webdav/") FINISHED WITH STATUS "AuthenticationRequiredError Error transferring https://cloud.XXXXX.com/remote.php/webdav/ - server replied: "
[OCC::PropfindJob::finished 	*not* successful, http result code is 401 ""
[OCC::WebFlowCredentials::stillValid 	Still valid?
[OCC::WebFlowCredentials::stillValid 	QNetworkReply::NetworkError(AuthenticationRequiredError)
[OCC::WebFlowCredentials::stillValid 	"Error transferring https://cloud.XXXXX.com/remote.php/webdav/ - server replied: "
````

**Comment by Florentin68** [comment]
Problem solved for me in daily version 2.5.0.61353 (build 20180905) ; I did not test other daily versions.

**Comment by splitt3r** [comment]
Looks good now :)
